<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pomodoro_Timer</class>
 <widget class="QWidget" name="Pomodoro_Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>292</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Pomodoro_Timer</string>
  </property>
  <widget class="QLCDNumber" name="hours_display">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="value" stdset="0">
    <double>24.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="minutes_display">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="value" stdset="0">
    <double>60.000000000000000</double>
   </property>
  </widget>
  <widget class="QLCDNumber" name="seconds_display">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="value" stdset="0">
    <double>60.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
   <property name="sliderPosition">
    <number>90</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
   <property name="notchTarget">
    <double>30.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="state_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>8514oem</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Short Break</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settings_button">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="notes_button">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Notes</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="notes_text">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:6pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Hello&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="settings_frame" name="settings_frame">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>40</y>
     <width>76</width>
     <height>95</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>settings_frame</class>
   <extends>QFrame</extends>
   <header>settings_frame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
